dev/build/windows/makecoq_mingw.sh

```diff
@@ -983,6 +983,15 @@ function make_ocaml {
       cp Changes      "$PREFIXOCAML/license_readme/ocaml/Changes.txt"
     fi
 
+    # Since 4.07 this library is part of ocaml
```

Comment:
Isn't it a bug in OCaml that this is not added to site-lib?

Reply:
Sincerly I don't know. I think it is the opam packaging that adds this file, but I did not investigate.

Reply:
The odd thing is that I don't need this for any of the other many OCaml libs we use. So this is kind of weird and should be investigated. On the other hand we will switch to opam soon.